internal/agent/delta/store.go

```diff
@@ -358,7 +358,7 @@ func (d *Store) updateLastDeltaSent(entityKey string, delta *inventoryapi.RawDel
 			})
 			if resultHint != nil {
 				if resultHint.Error != nil {
-					dslog.WithFields(logrus.Fields{"error": *resultHint.Error}).
+					dslog.WithField("error", *resultHint.Error).
```

Comment:
WithError?

Reply:
It isn't an error object but a string here. At first I thought the same and then when I went to do `WithError` I noticed it wasn't actually of type `error`.